doc/development/api/virtual_schema_api.md

```diff
@@ -414,8 +414,10 @@ will produce the following Request, assuming that the Adapter has all required c
 Notes
 * `pushdownRequest`: Specification what needs to be pushed down. You can think of it like a parsed SQL statement.
   * `from`: The requested from clause. This can be a table or a join.
-  * `selectList`: The requested select list elements, a list of expression. The order of the selectlist elements matters. If the select list is an empty list, we request at least a single column/expression, which could also be constant TRUE.
-  * `selectList.columnNr`: Position of the column in the virtual table, starting with 0
```

Comment:
Does `columnNr` still exist in the API? Why do we remove this information?

Reply:
`columnNr` is already documented here: https://github.com/exasol/virtual-schema-common-java/blob/main/doc/development/api/virtual_schema_api.md#column-lookup
It's weird that we included that one specific member of a column lookup here, so I removed it.

Reply:
It's fine if it's documented in another place! Then I'll merge the branch